### PR TITLE
Show deleted items

### PR DESCRIPTION
**Pull request by TheoLeanse**
@katsuraku 

screen shot 2016-07-29 at 15 06 51

### CODE REVIEW

**Comment by ookejimift** [comment]
Is it worth enabling click on deleted items from the search page as well

**Comment by TheoLeanse** [comment]
@ookejimift good call - done!

File: client/components/redirectManagement.js [hunk 1]
```diff
@@ -1,18 +1,25 @@
...
+export default ({ vanity, nonVanities, create }) => {
 	let fromURL;
 	return (
 		<div className="ums-manage-section">
-			<h3 className="ums-manage-heading">Redirects:</h3>
+			<h3 className="ums-manage-heading">Redirects (and more):</h3>
+
+			{ nonVanities.map((nonVanity, i) => {
+
+				return nonVanity.get('code') <= 302 ?
```
Comment (katsuraku) [inline client/components/redirectManagement.js]: ;D

**Comment by katsuraku** [comment]
👍

**Pull request merged**